marklogic-data-hub/src/main/java/com/marklogic/hub/scaffold/Scaffolding.java

```diff
@@ -42,29 +54,96 @@ static String getAbsolutePath(String first, String... more) {
         return absolutePath.toString();
     }
 
+    /**
+     * Returns the directory of the flow
+     * @param entityName - the entity name
+     * @param flowName - the name of the flow
+     * @param flowType - FlowType (sjs or xqy)
+     * @return the directory path as a Path type
+     */
```

Comment:
look at where this bad boy is being used. Mostly for internal and testing. Only 1 instance in quickstart that is questionable, at best.

Reply:
Going to leave this in due to quickstart until we can figure out a better way to support the quickstart use without invoking a direct implementation.